docs/pages/deploy-a-cluster/deployments/aws-starter-cluster-terraform.mdx

```diff
@@ -0,0 +1,714 @@
+---
+title: Teleport Single Instance deployment on AWS
+description: How to quickly configure Teleport on a single instance for testing in AWS.
+tocDepth: 3
+---
+
+This guide is designed to accompany our [reference starter-cluster Terraform code](https://github.com/gravitational/teleport/tree/branch/v(=teleport.major_version=)/examples/aws/terraform/starter-cluster#teleport-terraform-aws-ami-simple-example)
+and describe how to manage the resulting Teleport deployment.
+
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+## Prerequisites
+
+Our code requires Terraform 0.13+. You can [download Terraform here](https://www.terraform.io/downloads.html). We will assume that you have
+`terraform` installed and available on your path.
+
+```code
+$ which terraform
+/usr/local/bin/terraform
+$ terraform version
+Terraform v1.5.6
+```
+
+You will also require the `aws` command line tool. This is available in Ubuntu/Debian/Fedora/CentOS and MacOS Homebrew
+as the `awscli` package.
+
+Fedora/CentOS: `yum -y install awscli`
+
+Ubuntu/Debian: `apt-get -y install awscli`
+
+MacOS (with [Homebrew](https://brew.sh/)): `brew install awscli`
+
+When possible, installing via a package is always preferable. If you can't find a package available for
+your distribution, you can also download the tool from [https://aws.amazon.com/cli/](https://aws.amazon.com/cli/)
+
+We will assume that you have configured your AWS cli access with credentials available at `~/.aws/credentials`:
```

Comment:
```suggestion
We will assume that you have configured your AWS CLI access with credentials available at `~/.aws/credentials`:
```

Reply:
We already state this 20 lines below when we talk about `awscli` - was this intentional?